tensorflow/python/distribute/distribute_lib.py

```diff
@@ -937,13 +937,15 @@ def scope(self):
   # pylint: enable=line-too-long
 
   @doc_controls.do_not_doc_inheritable  # DEPRECATED, moving to `extended`
+  @deprecated(None,'use extended.colocate_vars_with() instead.')
   def colocate_vars_with(self, colocate_with_variable):
     """DEPRECATED: use extended.colocate_vars_with() instead."""
     return self._extended.colocate_vars_with(colocate_with_variable)
 
   @doc_controls.do_not_generate_docs  # DEPRECATED: TF 1.x only
   def make_dataset_iterator(self, dataset):
     """DEPRECATED TF 1.x ONLY."""
+    
```

Comment:
```suggestion
```

Reply:
Resolved. And added deprecation decorators where-ever specified.